### ISSUE TITLE
Support for self-hosted git servers

### ISSUE THREAD

**Issue by dmitriym09** [Rich-Harris/degit#173]
I want to add the ability to use self-hosted servers. I want to realize this through agrument **--trusted / -t**, such as:
```bash
degit git@my.gitlab.io:user/repo#dev my-app -t my.gitlab.io
```
Do you need it in the master or is it better to make my fork and keep it?

Thank you!

**Comment by keerthivasan-r** [comment]
Sounds like a good one. Are you working on this one anyway?

**Comment by dmitriym09** [comment]
> Sounds like a good one. Are you working on this one anyway?

Yes, I already did it. I want to create a PR if it is merge in the master branch.

**Comment by edjw** [comment]
Did anything happen with this? Did you create the PR @dmitriym09 ?

**Comment by dohooo** [comment]
You can try the new library degitlab.

**Comment by edjw** [comment]
Thank @dohooo https://github.com/dohooo/degitlab

**Comment by YogliB** [comment]
Hey @dmitriym09 - self-hosted GitLab is now supported via `gitlab://host/user/repo` (added in #456, documented in the help text). It doesn't use the `--trusted`/`-t` flag, but the new syntax should cover the same use case. Thanks for pushing for this!
